integ/simple_test.go

```diff
@@ -208,19 +195,10 @@ func (s *IntegSuite) TestSimple(c *C) {
 		log.Info("Attempting to roll forward with fix")
 
 		// Fixing update
-		var TaskManagerFixedResources = corev1.ResourceRequirements{
-			Requests: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("0.2"),
-				corev1.ResourceMemory: resource.MustParse("200Mi"),
-			},
-			Limits: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("0.2"),
-				corev1.ResourceMemory: resource.MustParse("200Mi"),
-			},
-		}
 		// and we should be able to roll forward by resubmitting with a fixed config
 		updateAndValidate(c, s, config.Name, func(app *v1beta1.FlinkApplication) {
-			app.Spec.TaskManagerConfig.Resources = &TaskManagerFixedResources
+			app.Spec.Image = NewImage
```

Comment:
This is so much better. Apologies for causing issues with the previous version of the test. :(